src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.1_9.cs

```diff
@@ -1,8 +1,7 @@
-#if MESSAGEPACK_1_9
 using System;
```

Comment:
Let's rename this file to `SpanMessagePackFormatter.cs` since we're dropping 2.1.

Reply:
Done in 80ecc20c831ce9afc1dd335d0c13884ce82a7fd5